examples/deploy-fly/README.md

```diff
@@ -0,0 +1,42 @@
+# Deploy Ollama to Fly.io
+
+## Prerequisites
+
+- Ollama: https://ollama.ai/download
+- Fly.io account. Sign up for a free account: https://fly.io/app/sign-up
+
+## Steps
+
+1. Login to Fly.io: `fly auth login`
+1. Create a new Fly app: `fly launch --name <name> --internal-port 11434 --vm-size shared-cpu-8x --yes`
```

Comment:
Seems like this is missing a step of deploying Ollama on the fly.io instance?

Reply:
That means they need to run the commands from the `examples/deploy-fly` directory correct? I think we should mention that.

Reply:
None of the other examples explicitly mentions this despite similar requirements